### PR TITLE
 proposing nonconflict comain

### PR DESCRIPTION
**Pull request by mchf**
Bnc#922765

### CODE REVIEW

**Comment by mvidner** [comment]
s/comain/Cobain/ ! :)

**Comment by mchf** [comment]
I'm thinking of kodein ;-)

File: package/yast2-network.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Jun  5 08:21:01 UTC 2015 - mfilka@suse.com
+
+- bnc#923990
+  - replaced already registered ".site" TLD using ".suse"
```
Comment (mvidner) [inline package/yast2-network.changes]: > replaced the already registered ".site" TLD with ".suse"

**Comment by mvidner** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for review.

**Pull request merged**